### PR TITLE
Hotfix/user management module

### PR DESCRIPTION
**Pull request by bestworks221**
## Description
User Management Module
Fixed functionality, api and added action on the settings

## Types of Changes

- [ ] Fixed issue on saving a new user
- [ ] Fixed issue on displaying undefined last name
- [ ] Fixed issue on updating account info
- [ ] Fixed issue on saving boolean value for Communication
- [ ] Fixed API issue on updating password
- [ ] Added functionality on user settings tab

### CODE REVIEW

File: resources/js/pages/admin/user/create/Submission.vue [hunk 1]
```diff
@@ -97,7 +97,7 @@ export default {
     },
 
     mounted() {
-
+        console.log(this.form)
```
Comment (ri-bulatao) [inline resources/js/pages/admin/user/create/Submission.vue:100]: Remove this Console log.

File: resources/js/pages/admin/user/edit/Settings.vue [hunk 2]
```diff
@@ -1,152 +1,182 @@
...
+            settings: 'email_settings/settings'
+        }),
 
-export default {
-    name: 'user.edit.settings',
+        created() {
+            // Fetch the email settings.
+            this.$store.dispatch('email_settings/fetchEmailSettings').then(()=> {
+                // Fill the form with email settings data.
+                console.log(this.settings)
```
Comment (ri-bulatao) [inline resources/js/pages/admin/user/edit/Settings.vue:159]: Remove Console Log

**Review by ri-bulatao — CHANGES_REQUESTED** [review]

File: resources/js/pages/admin/user/create/Submission.vue (hunk 1 above)
Comment (bestworks221) [reply resources/js/pages/admin/user/create/Submission.vue]: Removed Console log

File: resources/js/pages/admin/user/edit/Settings.vue (hunk 2 above)
Comment (bestworks221) [reply resources/js/pages/admin/user/edit/Settings.vue]: Removed Console Log

**Review by ri-bulatao — APPROVED** [review]
LGTM!

**Pull request merged**